docs/compilers/CSharp/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -364,3 +364,40 @@ class extension { } // type may not be named "extension"
 class C<extension> { } // type parameter may not be named "extension"
 ```
 
+## Partial properties and events are now implicitly virtual and public
+
+***Introduced in Visual Studio 2022 version 17.15***
+
+We have fixed [an inconsistency](https://github.com/dotnet/roslyn/issues/77346)
+where partial interface properties and events would not be implicitly `virtual` and `public` unlike their non-partial equivalents.
+This inconsistency is however [preserved](./Deviations%20from%20Standard.md#interface-partial-methods) for partial interface methods to avoid a larger breaking change.
+Note that Visual Basic and other languages not supporting default interface members will start requiring to implement implicitly virtual `partial` interface members.
+
+To keep the previous behavior, explicitly mark `partial` interface members as `sealed` and `private`.
```

Comment:
nit: `private` by itself should accomplish this, right? Since the member will be implicitly `sealed` in that case. It looks like `private sealed` is permitted on static interface methods but not instance ones.

Reply:
`private` will be enough for members like `partial int P { get; }`. But if you had a member like `public partial int P { get; }`, the workaround to preserve previous behavior is adding `sealed` modifier (you cannot add `private` since the member is already `public`). Let me try to clarify, thanks.